docker/nginx/fastcgi.conf

```diff
@@ -0,0 +1,21 @@
+fastcgi_param  SCRIPT_FILENAME    $document_root$fastcgi_script_name;
```

Comment:
This seems like mostly the default config? In that case you don't need to specify it.

Reply:
It indeed is default. Not sure if I had to specify it then, but we can leave it in I guess